homeassistant/components/twinkly/light.py

```diff
@@ -237,7 +241,7 @@ async def async_turn_on(self, **kwargs: Any) -> None:
 
             self._attr_rgb_color = kwargs[ATTR_RGB_COLOR]
 
-        if (
+        elif (
```

Comment:
Same with *brightness* and *effect*.  They can both be set in the same turn_on().

*Color* and *effect* would be mutually exclusive as they set different modes, but then we would need to prioritise using the principle of least surprise.  What should happen if both color and effect is set in the same turn_on()?